### DISCUSSION TITLE
location validation problem

### DISCUSSION THREAD

**Discussion by Awesore** [community/community#148033] [GitHub Education]
### Hi everyone,

After I selected the university and clicked, “continue” I have the button changed color, but nothing happened (yes I give access to my location).
No, I don't have my VPN turned on.

**Comment by davevad93** [comment]
Hi @Awesore ,

The problem you describe seems to be more browser related, when it's a location problem it usually returns an error like `You appear not to be near any campus location for the school you have selected.`. Try logging out from the Education portal, clear your browser cache (or use another browser or device) and try again.
